data/planners/254c7035-de7d-4d76-a888-2c09ba594eca.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+id: 254c7035-de7d-4d76-a888-2c09ba594eca.yml
```

Comment:
The `.yml` is not required for planners to register with the `data_svc`.

Reply:
That is good to know. In this case, we have some ability rewards in the .yml file, so I assume we still need the this?